src/Zynga/Framework/Cache/V2/Driver/Memcache.hh

```diff
@@ -247,4 +289,9 @@ class Memcache extends DriverBase {
 
   }
 
+  public function close(): bool {
```

Comment:
Don't think you need this. The connections created by add server are persistent connections unless you specify otherwise. 
Add server - https://www.php.net/manual/en/memcache.addserver.php
https://www.php.net/manual/en/memcache.close.php